nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/provenance.css

```diff
@@ -178,31 +178,26 @@ div.flowfile-container {
     width: 282px;
     height: 120px;
```

Comment:
@scottyaslan Also please add `line-height:1.4`. When multiples are listed they sit a little close to each other.